src/mesh/mesh.cpp

```diff
@@ -1326,4 +1248,15 @@ int Mesh::ReserveTagPhysIDs(int num_phys) {
 
 void Mesh::ReserveMeshBlockPhysIDs() { return; }
 
+std::int64_t Mesh::GetTotalCells() {
+  auto &pmb = block_list.front();
+  return static_cast<std::int64_t>(nbtotal) * pmb->block_size.nx1 * pmb->block_size.nx2 *
+         pmb->block_size.nx3;
+}
+// TODO(JMM): Move block_size into mesh.
```

Comment:
TODO can go away now presumably

Reply:
Should it? Seems like `block_size` is still on `MeshBlock`.

Reply:
oh I thought it moved. Nevermind.